portfolio/src/main/webapp/script.js

```diff
@@ -54,32 +36,72 @@ function addRandomATLAQuote() {
 }
 
 function addRandomMusicRecommendation(){
-    // Create arrays for song titles and their track images; both are related by indices
-    const songTitles = ['"telepatía" - Kali Uchis',
-                        '"She" - Harry Styles',
-                        '"Adiós" - Selena Gomez',
-                        '"Call My Friends" - Shawn Mendes',
-                        '"My Hair" - Ariana Grande'];
-    const songImages = ['/images/telepatia.jpg', 
-                        '/images/She.jpg',
-                        '/images/Adios.png',
-                        '/images/CallMyFriends.JPG',
-                        '/images/MyHair.JPG'];
+    // Create an array of song objects that contain the following information about a song: title, artist, album it comes from, release date, the song image url, the youtube link to play the song
+    const songs = [{
+                            songTitle: 'telepatía',
+                            artistName: 'Kali Uchis',
+                            songAlbum: 'Sin Miedo (del Amor y Otros Demonios)',
+                            yearReleased: '2020',
+                            albumCoverURL: '/images/telepatia.jpg',
+                            youtubeLinkToSong: 'https://www.youtube.com/watch?v=Dwzk-XZxZ4k'
+                        },
+                        {
+                            songTitle: 'She',
+                            artistName: 'Harry Styles',
+                            songAlbum: 'Fine Line',
+                            yearReleased: '2019',
+                            albumCoverURL: '/images/She.jpg',
+                            youtubeLinkToSong: 'https://www.youtube.com/watch?v=zQ3PeDGswz4'
+                        },
+                        {
+                            songTitle: 'Adiós',
+                            artistName: 'Selena Gomez',
+                            songAlbum: 'Revelación',
+                            yearReleased: '2021',
+                            albumCoverURL: '/images/Adios.png',
+                            youtubeLinkToSong: 'https://www.youtube.com/watch?v=9H_368c2Hzw'
+                        },
+                        {
+                            songTitle: 'Call My Friends',
+                            artistName: 'Shawn Mendes',
+                            songAlbum: 'Wonder',
+                            yearReleased: '2020',
+                            albumCoverURL: '/images/CallMyFriends.JPG',
+                            youtubeLinkToSong: 'https://www.youtube.com/watch?v=rpFjfpBZ-qA'
+                        },
+                        {
+                            songTitle: 'my hair',
+                            artistName: 'Ariana Grande',
+                            songAlbum: 'positions',
+                            yearReleased: '2020',
+                            albumCoverURL: '/images/MyHair.JPG',
+                            youtubeLinkToSong: 'https://www.youtube.com/watch?v=Sujm6756pZU'
```

Comment:
BTW I just realized this (you don't need to change this), but did you know you can embed youtube videos in your page?

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/Sujm6756pZU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Looks like the `src=` here is just the video ID like `Sujm6756pZU` for Ariana Grandes video.

Reply:
Oh that's wicked! Updated the portfolio to add this in :)